install_config/aggregate_logging.adoc

```diff
@@ -780,11 +780,150 @@ ifdef::openshift-enterprise[]
+====
+endif::openshift-enterprise[]
+
+Curator allows administrators to configure scheduled Elasticsearch maintenance operations
+to be performed automatically on a per-project basis. It is scheduled to run every
+day within its pod and will perform actions based on its configuration and recommended
+that only one pod be deployer per Elasticsearch cluster. Curator is configured
+via a mounted yaml configuration file in the following structure:
+
+====
+----
+$PROJECT_NAME:
+  $ACTION:
+    $UNIT: $VALUE
+
+$PROJECT_NAME:
+  $ACTION:
+    $UNIT: $VALUE
+ ...
+
+----
+====
+
+The available parameters are:
+
+[cols="3,7",options="header"]
+|===
+|Variable Name
+|Description
+
+|`*$PROJECT_NAME*`
+|The actual name of a project, such as `myapp-devel`.  For {product-title} `operations`
+logs, use the name `.operations` as the project name.
+
+|`*$ACTION*`
+|The action to take, currently only `delete` is allowed.
+
+|`*$UNIT*`
+|One of `days`, `weeks`, or `months`.
+
+|`*$VALUE*`
+|An integer for the number of units.
+
+|`*.defaults*`
+|Use `.defaults` as the `$PROJECT_NAME` to set the defaults for projects that are
+not specified.
+
+|`*runhour*`
+|(Number) the hour of the day in 24-hour format at which to run the Curator jobs. For
+use with `.defaults`.
+
+|`*runminute*`
+|(Number) the minute of the hour at which to run the Curator jobs. For use with `.defaults`.
+|===
+
+For example, to configure Curator to
+
+- delete indices in the `myapp-dev` project older than `1 day`
+- delete indices in the `myapp-qe` project older than `1 week`
+- delete `operations` logs older than `8 weeks`
+- delete all other projects indices after they are `30 days` old
+- run the Curator jobs at midnight every day
+
+you would use:
+
+----
+myapp-dev:
+ delete:
+   days: 1
+
+myapp-qe:
+  delete:
+    weeks: 1
+
+.operations:
+  delete:
+    weeks: 8
+
+.defaults:
+  delete:
+    days: 30
+  runhour: 0
+  runminute: 0
+----
+
+
+[IMPORTANT]
+====
+When you use `month` as the `$UNIT` for an operation, Curator starts counting at
+the first day of the current month, not the current day of the current month.
+For example, if today is April 15, and you want to delete indices that are 2 months
+older than today (delete: months: 2), Curator does not delete indices that are dated
+older than February 15; it deletes indices older than February 1. That is, it
+goes back to the first day of the current month, then goes back two whole months
+from that date. If you want to be exact with Curator, it is best to use days
+(e.g. `delete: days: 30`)
+====
+
+=== Creating the Curator configuration
+
+To create the Curator configuration, first create a yaml file with your configuration
+settings using your favorite editor.
+
+Next, create a secret from your created yaml file:
+----
+oc secrets new index-management settings=</path/to/your/yaml/file>
+----
+
+Then, mount your created secret as a volume in your Curator DC:
```

Comment:
Origin uses a configmap, and it's also mounted at a different place (I wanted the configmap next to the keys, instead of a directory higher). This may make for a bit of a snafu that we explain away by saying "3.2.1 was tech preview".